Modules/_asynciomodule.c

```diff
@@ -185,6 +185,12 @@ typedef struct {
     /* Counter for autogenerated Task names */
     uint64_t task_name_counter;
 
+#ifdef MS_WINDOWS
+    /* Pointer to the asyncio debug offset to avoid it to be optimized away
+       by the compiler in Windows (other platforms don't need this) */
+    void *debug_offsets;
+#endif
```

Comment:
```suggestion
    /* Pointer to the asyncio debug offset to avoid it to be optimized away
       by the compiler in Windows (other platforms don't need this) */
    void *debug_offsets;
```